src/util.h

```diff
@@ -124,6 +125,12 @@ namespace palace
     return attributes;
   }
 
+  inline alure::FilterParams
+  make_filter_params (std::vector<float> params) noexcept
+  {
+    return alure::FilterParams {params[0], params[1], params[2]};
+  }
```

Comment:
You might want to join the three lines together, otherwise it's OK to squash and merge now.